### ISSUE TITLE
exclude disabled activities from ics export

### ISSUE THREAD

**Issue by amengsk** [yunity/karrot-frontend#2454]
activities which are disabled in karrot should be either
1/ excluded from the ics export
2/ included with the "STATUS=CANCELED" property. other events should have "STATUS=CONFIRMED"

i think i prefer 2/

**Comment by nicksellen** [comment]
Yes 2) sounds good to me, without knowing anything about how apps handle the statuses...
